R/model_LaTeX.R

```diff
@@ -88,7 +102,7 @@ SEIRD_LaTeX <- function(mu) {
       r"(Recovered $$\frac{dR}{dt} = \sigma I - \mu_D R $$)",
       r"(Dead $$ \frac{dD}{dt} = \delta I $$)",
       r"(Reproductive ratio $$R_0 =  \frac{\beta}{\gamma}$$)",
-      "q-Value", br(), r"($$ 1, frequency-dependent $$)", br(), r"($$ 0, density-dependent $$)"
+      "q-Value $$\\begin{equation} x = \\begin{cases} 1 & \\text{, } frequency − dependent \\\\ 0 & \\text{, } density − dependent \\end{cases}\\end{equation} $$"
```

Comment:
Please use raw strings when writing $\LaTeX{}$. Inside the raw string you will only have to write $\LaTeX{}$, and won't need to escape backslash characters which are used to call macros and commands.